CHANGES

```diff
@@ -9,6 +9,14 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Added functionality to create an EVP_PKEY context based on data
```

Comment:
This seems like a confusing description here. The API docs are clearer than this.

Reply:
Uhmmmmm...  this description has nothing to do with this PR, it belongs with #10184...  did I mix the CHANGES changes up? ...  yup, this seems to be a rebasse goof-up

Reply:
Fixed